### PR TITLE
VY-78 Create Employee Detail page

### PR DESCRIPTION
**Pull request by lf-kiranpariyar**
@lf-romitamgai . Please review the code.

### CODE REVIEW

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

**Comment by bhuwang** [comment]
Can one of the admins verify this patch?

File: frontend/src/js/components/employee/employee-view/SocialAccountsInfo.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  constructor(props) {
+    super(props);
+    this.renderSocialAccountList = this.renderSocialAccountList.bind(this);
+  }
+
+  renderSocialAccountList(account,key) {
+    let accountExtension = account.split('@');
+    let fontClass = '';
+
+    if(accountExtension[1] == 'skype.com') {
```
Comment (lf-romitamgai) [inline frontend/src/js/components/employee/employee-view/SocialAccountsInfo.js]: use switch instead

File: frontend/src/css/main.css [hunk 2]
```diff
@@ -1399,3 +1399,108 @@ ul.pagination li {
...
+.employee-detail-right{
+    border-left: 1px solid #bdbdbd;
+}
+
+.employee-detail-left{
+    border-right: 1px solid #bdbdbd;
+}
+
+.employee-img-div{
+
```
Comment (joshianuj) [inline frontend/src/css/main.css]: empty style

File: frontend/src/js/components/employee/employee-view/EmployeeInfoTab.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+import {Tab, Tabs, TabList, TabPanel} from 'react-tabs';
+
+class EmployeeInfoTab extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleSelect(index, last) {
```
Comment (joshianuj) [inline frontend/src/js/components/employee/employee-view/EmployeeInfoTab.js]: Add TO:DO if it's a pending task?

**Pull request merged**